cms-adapter/src/Rsbc.Dmf.CaseManagement.Service/Services/CaseService.cs

```diff
@@ -1665,8 +1690,18 @@ public async override Task<SearchReply> Search(SearchRequest request, ServerCall
                     };
                     newCase.Flags.Add(c.Flags.Select(f => new FlagItem
                     {
-                        Identifier = f.Id, Question = f.Description ?? "Unknown", FlagType = ConvertFlagType(f.FlagType)
+                        Identifier = f.Id,
+                        Question = f.Description ?? "Unknown",
+                        FlagType = ConvertFlagType(f.FlagType),
+                        FormId = f.FormId ?? ""
                     }));
+                    newCase.MedicalConditions.Add(
```

Comment:
note to self: Search is used in 5 other places in legacy apps, will need to regression test these changes